clang/include/clang/Sema/Sema.h

```diff
@@ -12884,6 +12884,11 @@ class Sema final {
   void MarkDevice();
   void MarkSyclSimd();
 
+  /// Diagnoses an attribute in the 'intelfpga' namespace and suggests using
+  /// the attribute in the 'intel' namespace instead.
+  void CheckDeprecatedSYCLAttributeSpelling(const ParsedAttr &Attr,
```

Comment:
I guess it doesn't matter as much in function declaration but should `Attr `be `A `here as well?

Reply:
Yup, good catch! I'll correct that and push a new patch.